autorest/client.go

```diff
@@ -230,6 +231,12 @@ func (c Client) Do(r *http.Request) (*http.Response, error) {
 func (c Client) sender() Sender {
 	if c.Sender == nil {
 		j, _ := cookiejar.New(nil)
+		tracing.Transport.Base = &http.Transport{
+			TLSClientConfig: &tls.Config{
+				MinVersion:               tls.VersionTLS12,
+				PreferServerCipherSuites: true,
```

Comment:
hmm the `preferServerCipherSuites` has any effect on clients ? AFAIK this setting is usually for servers to use their own cipher suites as opposed to what they receive from clients.

Reply:
Hmm it could be that I misread [this article](https://www.socketloop.com/tutorials/golang-use-tls-version-1-2-and-enforce-server-security-configuration-over-client).  I can remove this if you agree.

Reply:
yea i think there its saying that when its configured on the server the clients are forced to use the servers cipherSuites

Reply:
Removed.